platform-includes/enriching-events/attach-screenshots/flutter.mdx

```diff
@@ -13,6 +13,32 @@ await SentryFlutter.init(
 );
 ```
 
+### Customize Screenshot Capturing
+
+<Note>
+
+Requires SDK version `8.11.0` or higher.
+
+</Note>
+
+Because capturing screenshots can be a resource-intensive operation on Flutter, it's limited to one screenshot every 2 seconds using a debouncing mechanism. This behavior can be overruled if you supply a `BeforeCaptureCallback` for screenshots in the `SentryFlutterOptions`.
+
+The `BeforeCaptureCallback` also allows you to customize the behavior based on event data, so you can decide when to capture a screenshot and when not to. For example, you can decide to only capture screenshots of fatal events:
+
+```java
```

Comment:
```suggestion
```dart
```

Reply:
also please add line highlighting

Reply:
Not sure if highlighting makes sense here, as it's pretty much the whole code block.